internal/command/resume/worker.go

```diff
@@ -0,0 +1,42 @@
+package resume
+
+import (
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func newResumeVMCommand() *cobra.Command {
```

Comment:
```suggestion
func newResumeWorkerCommand() *cobra.Command {
```

Reply:
Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.